<!doctype html>
<html lang="en">
  <head>
    <meta charset="utf-8">
    <meta name="viewport" content="width=device-width, initial-scale=1">
    <meta name="title" content="Novel antimalarial drug target identification | Katie Amberg-Johnson | Professional Portfolio">
    <meta name="description" content="">
    <meta name="image" content="https://github.com/kambergjohnson.png">
    <meta property="og:type" content="website">
    <meta property="og:site_name" content="Katie Amberg-Johnson | Professional Portfolio">
    <meta property="og:url" content="https://kambergjohnson.github.io/projects/antimalarial-target-identification.html">
    <meta property="og:title" content="Novel antimalarial drug target identification | Katie Amberg-Johnson | Professional Portfolio">
    <meta property="og:description" content="">
    <meta property="og:image" content="https://github.com/kambergjohnson.png">
    <link rel="shortcut icon" href="/favicon.ico">
    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.2.0-beta1/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-0evHe/X+R7YkIZDRvuzKMRqM+OrBnVFBL6DOitfPri4tjfHxaWutUpFmBp4vmVor" crossorigin="anonymous">
    <link rel="stylesheet" href="/css/techfolio-theme/default.css">
    <link rel="stylesheet" type="text/css" href="/css/rouge/github.css">
    <!-- Load MathJax if 'mathjax: true' is found in your _config.yml. -->
    
    <script type="text/javascript" async src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.7/latest.js?config=TeX-MML-AM_CHTML">
    </script>
    

    <title>Novel antimalarial drug target identification | Katie Amberg-Johnson | Professional Portfolio</title>
  </head>
  <body>
  <header class="navbar navbar-expand-md navbar-light bg-light bg-gradient border-bottom">
  <div class="container-fluid">
    <a class="navbar-brand" href="/">Katie Amberg-Johnson</a>
    <div class="ms-auto">
      <ul class="navbar-nav mb-2 mb-lg-0">
        <a class="nav-link" href="/#papers">Papers</a>
        <a class="nav-link" href="/#projects">Projects</a>
        <a class="nav-link" href="/about-me.html">About</a>
        <a class="nav-link" href="/resources/Amberg_Johnson_Katherine_CV.pdf">CV</a>
      </ul>
    </div>
  </div>
</header>

<div class="container py-4">
  <h1 class="display-4">Novel antimalarial drug target identification</h1>
 <p>To identify the target of actinonin, we initially  attempted to isolate actinonin-resistant P. falciparum but failed using multiple selection methods, including chemical mutagenesis, that successfully selected resistance against other compounds. Therefore, we turned to Toxoplasma gondii, a related apicomplexan parasite, which contains an apicoplast of the same evolutionary origin, because it is easier to grow to large numbers and to genetically modify.</p>

<p>Because actinonin disrupted apicoplast biogenesis in both T. gondii and P. falciparum, suggesting a common target, we selected actinonin-resistant T.gondii and submitted resistant strains for whole-genome sequencing. Five out of eight independently selected clones contained a single point mutation in FtsH1, an apicoplast membrane metalloprotease. To validate FtsH1 as the genetic mechanism of resistance, we replaced the endogenous FtsH1 locus with either a wild-type or mutant FtsH1 locus. The mutated FtsH1 strain resulted in resistance, strongly suggesting FtsH1 as the target of actinonin.</p>

<p>Lastly, we validated FtsH1 as the target of actinonin in P. falciparum. Using a FstH1 knock-down line, we  down-regulated FtsH1 expression and observed a 58-fold decrease in the actinonin EC50 upon knockdown of FtsH1 expression. To confirm that actinonin bound directly to FtsH1, we expressed and purified FtsH1 in vitro and found that actinonin inhibited protease activity of purified FtsH1.</p>

<h3 id="a-mutation-in-the-protease-domain-of-ftsh1-is-sufficient-to-confer-resistance-to-actinonin-in-t-gondii">A mutation in the protease domain of ftsH1 is sufficient to confer resistance to actinonin in <em>T. gondii</em></h3>

<div class="m-4" style="border:1px solid black">
<img class="img-fluid " src="/images/Figure-2a.png" alt="" />
<p class="p-3"><small>Representative images of the apicoplast of control and actinonin treated parasites 36 hours after infection. The apicoplast is visualized using the T. gondii reporter strain RH FNR-RFP in which RFP is targeted to the apicoplast and the nucleus is stained with Hoescht 33342. Each parasite contains one apicoplast, except during cell division when there may be two. White arrows point at examples of T. gondii parasites missing an apicoplast.</small></p>
</div>

<div class="m-4" style="border:1px solid black">
<img class="img-fluid " src="/images/Figure-2b.png" alt="" />
<p class="p-3"><small>Dose-dependent parasite growth inhibition upon treatment with actinonin for the actinonin-sensitive parent strain (RH) compared with 3 independent clones following selection for actinonin resistance (resistant 1, resistant 2, resistant 3). These three resistant clones are representative of the eight clones submitted for whole genome sequencing. Growth was measured via summed areas of the plaques formed during plaque assays and normalized to untreated controls. Error bars represent the SEM of two biological replicates.</small></p>
</div>

<div class="m-4" style="border:1px solid black">
<img class="img-fluid " src="/images/Figure-2c.png" alt="" />
<p class="p-3"><small>Dose-dependent parasite growth inhibition upon treatment with actinonin for ftsH1WT compared with ftsH1(N805S) parasites in RH Î”ku80 strain. Data was measured and analyzed as in B.</small></p>
</div>

<div class="m-4" style="border:1px solid black">
<img class="img-fluid " src="/images/Figure-2d.png" alt="" />
<p class="p-3"><small>Schematic of TgFtsH1. This protein contains a N-unique region containing a putative transmembrane domain, an AAA ATPase domain used for unfolding proteins, a peptidase domain with a zinc co-factor in the catalytic site, and a C-unique region. The resistance-conferring variant FtsH(N805S) is found in the peptidase domain near the catalytic site.</small></p>
</div>

<h3 id="knockdown-of-ftsh1-in-p-falciparum-leads-to-apicoplast-loss-and-hypersensitivity-to-actinonin">Knockdown of ftsH1 in <em>P. falciparum</em> leads to apicoplast loss and hypersensitivity to actinonin</h3>

<div class="m-4" style="border:1px solid black">
<img class="img-fluid " src="/images/Figure-3a.png" alt="" />
<p class="p-3"><small>Schematic of the endogenous knockdown strategy. When aTC is present in the media, the tet-repressor binds aTC and does not bind the 10x-aptamer sequence, which relieves translational repression, allowing PfFtsH1 to be expressed. When aTC is washed out of the media, the tet-repressor binds the 10x-aptamer and prevents expression of PfFtsH1.</small></p>
</div>

<div class="m-4" style="border:1px solid black">
<img class="img-fluid " src="/images/Figure-3b.png" alt="" />
<p class="p-3"><small>Time course of parasite growth without aTC and in the presence or absence of IPP in the media, normalized to the untreated or IPP-rescued parental strain as appropriate. Error bars represent the SEM of two biological replicates.</small></p>
</div>

<div class="m-4" style="border:1px solid black">
<img class="img-fluid " src="/images/Figure-3c.png" alt="" />
<p class="p-3"><small>Time course of the apicoplast:nuclear genome ratio measured by quantitative PCR (qPCR) using primers for the apicoplast and nuclear genomes during treatment with or without aTC. All samples contained IPP to rescue parasite growth. Genome ratios were normalized to respective parental cultures also grown with IPP. Error bars as in C.</small></p>
</div>

<div class="m-4" style="border:1px solid black">
<img class="img-fluid " src="/images/Figure-3d.png" alt="" />
<p class="p-3"><small>Dose-dependent parasite growth inhibition by actinonin in the absence or presence of aTC. Error bars as in C.</small></p>
</div>

<h3 id="actinonin-inhibits-pfftsh1-in-vitro">Actinonin inhibits PfFtsH1 in vitro</h3>

<div class="m-4" style="border:1px solid black">
<img class="img-fluid " src="/images/Figure-4a.png" alt="" />
<p class="p-3"><small>Schematic of PfFtsH1 constructs used for biochemical assays. Amino acids 91-612 of the endogenous protein (PfFtsH1 endogenous), which include the AAA+ ATPase and peptidase domains, were placed between His6-SUMO and GST domains to aid in purification and solubility. WT is the parent construct, E249Q is an inactivating mutation in the AAA+ ATPase domain, and D493A is an inactivating mutation in the peptidase domain.</small></p>
</div>

<div class="m-4" style="border:1px solid black">
<img class="img-fluid " src="/images/Figure-4b.png" alt="" />
<p class="p-3"><small>ATP hydrolysis by PfFtsH1 WT and E249Q measured using a coupled spectrophotometric assay<sup>80</sup>.</small></p>
</div>

<div class="m-4" style="border:1px solid black">
<img class="img-fluid " src="/images/Figure-4c.png" alt="" />
<p class="p-3"><small>ATP-dependent proteolysis of FITC-labeled casein by PfFtsH1 WT and D493A
</small></p>
</div>

<div class="m-4" style="border:1px solid black">
<img class="img-fluid " src="/images/Figure-4d.png" alt="" />
<p class="p-3"><small>Dose-dependent parasite growth inhibition by actinonin (black) or actinamide (red) with and without knockdown of PfFtsH1. Error bars represent the SEM of two biological replicates.</small></p>
</div>

<div class="m-4" style="border:1px solid black">
<img class="img-fluid " src="/images/Figure-4e.png" alt="" />
<p class="p-3"><small>Dose-dependent proteolytic inhibition of FITC-labeled casein by PfFtsH1 WT. Error bars represent the SEM of 3 replicates.</small></p>
</div>


</div>

<footer class="navbar navbar-expand navbar-light bg-light bg-gradient border-top">
  <div class="container-fluid">
    <div class="ms-auto">
      <ul class="navbar-nav mb-2 mb-lg-0">
        <small><a class="nav-link" href="https://techfolios.github.io">Made with Techfolios</a></small>
      </ul>
    </div>
  </div>
</footer>


  <script src="https://cdn.jsdelivr.net/npm/bootstrap@5.2.0-beta1/dist/js/bootstrap.bundle.min.js" integrity="sha384-pprn3073KE6tl6bjs2QrFaJGz5/SUsLqktiwsUTF55Jfv3qYSDhgCecCxMW52nD2" crossorigin="anonymous"></script>

  <!-- Load Tableau visualization code for the beach contamination page -->
  <script src="https://public.tableau.com/javascripts/api/tableau-2.2.1.js"></script>

  <script>
    function initializeViz(element, url) {
      // JS object that points at empty div in the html
      var placeholderDiv = document.getElementById(element);
      if (placeholderDiv) {
        var options = {
          hideTabs: true,
          hideToolbar: true,
        };
        viz = new tableau.Viz(placeholderDiv, url, options);
      }
    }
    initializeViz("tableauViz", "https://public.tableau.com/views/waterqualitymap4/Dashboard1?:embed=y&:display_count=yes&publish=yes");
    initializeViz("tableauViz2", "https://public.tableau.com/views/waterqualityline/Dashboard2?:embed=y&:display_count=yes");
  </script>

  </body>
</html>
